plugins/search/src/components/SearchType/SearchType.test.tsx

```diff
@@ -31,7 +31,7 @@ describe('SearchType', () => {
     term: '',
     filters: {},
     types: [],
-    pageCursor: '',
+    page: {},
```

Comment:
would it be reasonable to try to unify our paging structures? https://github.com/backstage/backstage/blob/5a1eb6bfc44305711b28c2acc408a2c9f130bf32/plugins/catalog-backend/src/catalog/types.ts?plain=1#L24

Reply:
oh and come to think of it, the input query as well https://github.com/backstage/backstage/blob/5a1eb6bfc44305711b28c2acc408a2c9f130bf32/plugins/catalog-backend/src/service/request/parseEntityPaginationParams.ts?plain=1#L25

Reply:
Actually, that line was something is missed to remove - but is also unrelated to what you asked.

But anyway, we could change that. If I understand it correctly, the current scheme we use for the REST API is derived from GraphQL (or not? query parameters seem to work differently). Splitting out the `pageInfo` is probably a breaking change, but paging didn't worked till now anyway, so I don't see this as an issue. I would have to extend the `pageInfo` with a `startCursor` and `hasPreviousPage` to implement the back navigation. For the query parameters I would use `before`. I would not implement `offset` and `limit` as I think the whole point of using the cursor based model is to avoid them.

I will have a look and will give the PR an update.

Reply:
@freben After a second thought: I don't think that this change would be worth the effort. It wouldn't introduce any actually value (e.g. other usage patterns) compared to the current pattern and would cost me at least a day to change frontend, backend and all three search engines - time that I won't be able to invest right now. I already rewrote it once 😆 

If someone ones to pick this up, feel free to pull the branch in a new PR.